gravity-forms/gw-validate-that-a-value-exists.php

```diff
@@ -222,7 +222,7 @@ public function output_script() {
 									break;
 								}
 							}
-							values[ inputId ] = $( this ).val();
+							values[ sourceFieldId ] = $( this ).val();
```

Comment:
Is there a situation in which `sourceFieldId` could be undefined?

I'm wondering if it should be something like...

```suggestion
							values[ sourceFieldId ? sourceFieldId : inputId ] = $( this ).val();
```